packages/kit/src/runtime/server/index.js

```diff
@@ -295,7 +299,7 @@ export async function respond(request, options, state) {
 							response.headers.set(key, /** @type {string} */ (value));
 						}
 					}
-					add_cookies_to_headers(response.headers, Array.from(new_cookies.values()));
+					add_cookies_to_headers(response.headers, Array.from(Object.values(new_cookies)));
```

Comment:
```suggestion
					add_cookies_to_headers(response.headers, Object.values(new_cookies));
```
Since `Object.values()` returns `Array`

Reply:
d'oh. thanks!